bin/chrootBuddy.sh

```diff
@@ -24,6 +25,7 @@ while true; do
 		-r | --release) RELEASE=$2; shift 2;;
 		-a | --arch) ARCH=$2; shift 2;;
 		-P | --packages) PACKAGES_LIST+=" $2"; shift 2;;
+		-e | --extra) EXTRA_LIST=$2; shift 2;;
```

Comment:
no this extra parameter is optional. PKG_EXTRA is filled if --extra is provided. But indeed, there is a warning if extra is not set. This is going to be fixed.